whois-nrtm4/src/test/java/net/ripe/db/nrtm4/DeltaFileGenerationTestIntegration.java

```diff
@@ -321,7 +321,7 @@ public void snapshot_should_match_last_delta_version() throws JSONException {
         final UpdateNotificationFile testUpdateNotification = getNotificationFileBySource("TEST");
         final String[] testDelta = getDeltasFromUpdateNotificationBySource("TEST", 2);
 
-        assertThat(testUpdateNotification.getSnapshot().getVersion(), is(new JSONObject(testDelta[0]).getLong("version")));
+        assertThat(testUpdateNotification.getSnapshot().getVersion(), Matchers.is(new JSONObject(testDelta[0]).getLong("version")));
```

Comment:
same as before I think that it is better just to load the static method instead whole class...
This is like using "*" and load more that you really need

Reply:
Fixed